### PR TITLE
Update conda_forge_webservices

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: conda_forge_webservices/update_me.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import os
+from git import Repo, Actor
+
+from conda_build.conda_interface import (VersionOrder, MatchSpec,
+    get_installed_version, root_dir, get_index, Resolve)
+
+from .utils import tmp_directory
+
+
+def update_me():
```
Comment (CJ-Wright) [inline conda_forge_webservices/update_me.py]: would it be possible to add some docs here? (I guess this should be self evident but...)

**Comment by dougalsutherland** [comment]
Nice. Is the idea that this would just be pinged occasionally, or is there a way to hook it after builds are completed on `master` for conda-forge-pinning / conda-smithy?

**Comment by isuruf** [comment]
> is there a way to hook it after builds are completed on master for conda-forge-pinning / conda-smithy?

Yes, we can look at the circle-ci status.

File: conda_forge_webservices/update_me.py (hunk 1 above)
Comment (isuruf) [reply conda_forge_webservices/update_me.py]: sure

**Comment by jakirkham** [comment]
Thanks for working on this @isuruf. Was thinking this might be a reasonable way to solve the problem as well.

There is a [Status Event]( https://developer.github.com/v3/activity/events/types/#statusevent ) we could listen for updates too. We may also need to [list the statuses]( https://developer.github.com/v3/repos/statuses/#list-statuses-for-a-specific-ref ) after and check CircleCI passed. Though that shouldn't be needed if the Status Event has this information already.

**Pull request merged**